tools/script_template.py

```diff
@@ -0,0 +1,29 @@
+import alara_output_processing as aop
+import numpy as np
+import openmc
+import argparse
+import yaml
+import pandas as pd
+import sqlite3
```

Comment:
Let's add these as they become necessary
```suggestion
import argparse
import yaml
```